### ISSUE TITLE
Git Diff always returns no changes on WSL

### ISSUE THREAD

**Issue by KoceilaDRK** [continuedev/continue#2822]
### Before submitting your bug report

- [ ] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [ ] I've seen the [troubleshooting guide](https://docs.continue.dev/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
- OS: Windows 23H2 on WSL 2.3.24.0
- Continue version: v0.8.55
- IDE: VSCode 1.95.1
```

### Description

When I'm trying to use the @diff to try to add the git diff context, it alway return empty.

here's the content of the context `@Git diff` : 
`Git shows no current changes.`

doing the same command on a terminal shows the diff.

similar issue : #https://github.com/continuedev/continue/issues/1205

### To reproduce

1. Open a project on windows with WSL.
2. Make some change of the code
3. Open Continue and input @Git diff

### Log output

```Shell
nothing.
```

**Comment by vovacha** [comment]
Same.
```
- OS: MacOS Sequoia 15.1
- Continue version: v0.8.56
- IDE: VSCode 1.95.1
```

**Comment by sestinj** [comment]
Thanks for the report! Rather than trying to patch this particular error, we're going to take a step back and try to solve this and many of the other WSL problems at once in December. I'll report back when we have an update!

**Comment by geiseri** [comment]
I see this same behavior, but I am using a remote vscode.  Could it be that it is getting confused by a file location?

**Comment by roryeckel** [comment]
Agreed with the above, I'm using a VS Code remote as well and experiencing the issue of always "Git shows no current changes."

**Comment by HRworksArk** [comment]
I am facing this problem aswell. Continue 1.1.56

**Comment by vowless-dev** [comment]
Same issue on Continue 1.1.77

**Comment by NgrNxk** [comment]
I have this issue as well: continue 1.2.4 and WSL-Version: 2.5.9.0. Perhaps December this year will do? 🙈

**Comment by TecFancy** [comment]
> Thanks for the report! Rather than trying to patch this particular error, we're going to take a step back and try to solve this and many of the other WSL problems at once in December. I'll report back when we have an update!

It's already been a year, and this issue still hasn't been fixed ;-)
